src/commands/mod.rs

```diff
@@ -187,7 +192,18 @@ fn app() -> Command {
               .hide(true)
               .help("Always return 0 exit code."),
         )
-}
+        .subcommand(
+            Command::new("completions")
+            .about("Generate completions for the specified shell.")
+            .arg_required_else_help(true)
+            .arg(
+                Arg::new("shell")
+                    .long("shell")
+                    .help("The shell to print completions for.")
+                    .value_parser(value_parser!(Shell)),
```

Comment:
I think this would be nicer as a positional argument since it's the only possible argument and is also required. I.e.
```
sentry-cli completions fish
```
instead of
```
sentry-cli completions --shell fish
```

Reply:
Done. Didn't manage to use the `Shell` enum to avoid repetition for each shell without errors though, so I used a string for each shell.

Reply:
Sorry, what do you mean? Just deleting the `long` call seems to work fine.
```rust
            Command::new("completions")
            .about("Generate completions for the specified shell.")
            .arg_required_else_help(true)
            .arg(
                Arg::new("shell")
                    .help("The shell to print completions for.")
                    .value_parser(value_parser!(Shell)),
            )
```